python_wrapper/susi_python/models.py

```diff
@@ -136,6 +136,11 @@ def __init__(self, identifier , identifier_type):
         self.identifier = identifier
         self.identifier_type = identifier_type
 
+class MediaAction(BaseAction):
+    def __init__(self,mediaAction):
```

Comment:
Coding style

Reply:
@hongquan , what is wrong with the coding style?